code-analysis/crates/solidity/outputs/typescript_lib/.cargo/config.toml

```diff
@@ -0,0 +1,7 @@
+[target.aarch64-unknown-linux-gnu]
```

Comment:
are these files left over from experimentation? not sure I understand their relationship with the crates, or how they are configured this way.
Happy to talk about it F2F as well if I'm missing something.

Reply:
They are for building the xplatform napi lib.

The CI.yml is for when we build the napi native libs as part of our CI. Those files q